### PR TITLE
Leveldb integration

### PR DESCRIPTION
**Pull request by damirka**
Replaced `core/chaindata` and `core/pool` with new `core/db`.

Features:

- LevelDB inside
- potentially more features
- file-handling is no longer our task
- few bugs with streams fixed

### CODE REVIEW

File: clients/block-producer.js [hunk 1]
```diff
@@ -130,17 +132,23 @@ async function isMyRound(frn) {
         return true;
     }
 
+    console.log(block, typeof block);
```
Comment (borispovod) [inline clients/block-producer.js:133]: remove console.log

Comment (borispovod) [reply clients/block-producer.js:133]: or at least typeof output

File: core/db.js [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+ */
+class Pool extends DB {
+
+    /**
+     * Add transaction to the pool
+     *
+     * @param {String} tx Transaction to store
+     */
+    add(tx) {
+        return this.db.put(tx, tx);
```
Comment (borispovod) [inline core/db.js:78]: why not use transaction hash as key?

Comment (damirka) [reply core/db.js:78]: @borispovod 

1) There's no access to TX key in this part of app and it's not a DB logic to extract it.
2) Currently there's no reason to do it as pool is always drained and no BP would access pool's transactions by tx hash.

File: clients/block-producer.js (hunk 1 above)
Comment (damirka) [reply clients/block-producer.js:133]: It's gonna be removed anyway. Don't want to make full commit in a draft PR with only console.log removed. In next PR there won't be this file a'all.

**Review by borispovod — APPROVED** [review]

**Pull request merged**